src/hera.cpp

```diff
@@ -409,17 +419,10 @@ int hera_set_option(
   }
 
   if (strcmp(name, "engine") == 0) {
-     if (strcmp(value, "binaryen") == 0)
-       hera->wasm_engine = hera_wasm_engine::binaryen;
-#if HERA_WABT
-     if (strcmp(value, "wabt") == 0)
-       hera->wasm_engine = hera_wasm_engine::wabt;
-#endif
-#if HERA_WAVM
-     if (strcmp(value, "wavm") == 0)
-       hera->wasm_engine = hera_wasm_engine::wavm;
-#endif
-     return 1;
+    if (wasm_engine_names.count(name)) {
```

Comment:
I am pretty sure this should be `value`, not `name`.

Reply:
Fixed.